packages/contentstack-config/src/commands/config/set/rate-limit.ts

```diff
@@ -0,0 +1,80 @@
+import { flags, isAuthenticated, FlagInput, managementSDKClient, cliux } from '@contentstack/cli-utilities';
+import { RateLimitHandler } from '../../../utils/rate-limit-handler';
+import { BaseCommand } from '../../../base-command';
+import { askOrgID } from '../../../utils/interactive';
+
+interface RateLimitConfig {
+  org?: string;
+  utilize?: number;
+  limitName?: string[];
+  default?: boolean;
+  auth_token?: string;
+}
```

Comment:
@harshithad0703 ,
Since we have an interface folder, you can move the type definition there instead of keeping it within the command.
do we really need auth_token?